### ISSUE TITLE
Low & Wrong Accuracy of NASNet-A-Mobile_224

### ISSUE THREAD

**Issue by PineApple777** [wandering007/nasnet-pytorch#4]
I just use below model file and Use **ImageNet** (year: 2012)
The ported model files are provided: **NASNet-A_Mobile_224**
But, I got top 1 acc: 53.97%, top5: 32.50% and This seems overfitting.
Is there something settings should I check  when I run these things?

**Comment by PineApple777** [comment]
In your imagenet_eval.py code, Calculating topk maybe wrong.
I change like this to get top1 & top5

```python
_, predictions = output.data.topk(5, 1, True, True)
transpose_predictions = predictions.t()
target_reshaped = target.view(1, -1).expand_as(transpose_predictions)
topk_correct = (test_predictions == target_reshaped)
top1 += topk_correct[:1].reshape(-1).float().sum(dim=0, keepdim=True).item()
top5 += topk_correct[:5].reshape(-1).float().sum(dim=0, keepdim=True).item()
```
